### PR TITLE
Accessibility bug fixes

### PR DESCRIPTION
**Pull request by Lxiamail**
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1263107 
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1270529
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1059540

### CODE REVIEW

File: src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml [hunk 1]
```diff
@@ -25,7 +25,8 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
             <RowDefinition Height="Auto" />
             <RowDefinition />
         </Grid.RowDefinitions>
-        <TextBlock Text="{x:Static const:LocalizedStrings.AvailableReports}"
+        <TextBlock Name ="AvailabelReportsLabel"
```
Comment (marklio) [inline src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml:28]:
```suggestion
        <TextBlock Name="AvailableReportsLabel"
```

File: src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml [hunk 2]
```diff
@@ -36,7 +37,8 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
                           HorizontalScrollBarVisibility="Auto"
                           Background="Transparent"
                           IsTabStop="False">
-                <ListView ItemsSource="{Binding Paths}"
+                <ListView AutomationProperties.LabeledBy="{Binding ElementName=AvailabelReportsLabel}"
```
Comment (marklio) [inline src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml:40]:
```suggestion
                <ListView AutomationProperties.LabeledBy="{Binding ElementName=AvailableReportsLabel}" 
```

**Review by marklio — CHANGES_REQUESTED** [review]
Just a couple of nits here for consistency.

File: src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml (hunk 1 above)
Comment (Lxiamail) [reply src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml]: fixed

File: src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml (hunk 2 above)
Comment (Lxiamail) [reply src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml]: fixed

File: src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml [hunk 3]
```diff
@@ -25,7 +25,8 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
             <RowDefinition Height="Auto" />
             <RowDefinition />
         </Grid.RowDefinitions>
-        <TextBlock Text="{x:Static const:LocalizedStrings.AvailableReports}"
+        <TextBlock Name ="AvailableReportsLabel"
```
Comment (marklio) [inline src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml:28]: NIT: still a weird space here.
```suggestion
        <TextBlock Name="AvailableReportsLabel"
```

**Review by marklio — APPROVED** [review]

File: src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml [hunk 4]
```diff
@@ -83,14 +83,14 @@
         detailColumnHeaders.AddRange(Model.TargetHeaders);
         detailColumnHeaders.Add(LocalizedStrings.RecommendedChanges);
 
-        <h3>@LocalizedStrings.HideColumns</h3>
+        <h3 id="hidecolumnsId">@LocalizedStrings.HideColumns</h3>
         <form>
```
Comment (scottaohara) [inline src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml:87]:
```
<form>
  <fieldset aria-labelldby="hidecolumnsId">
</form>
```

File: src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml [hunk 5]
```diff
@@ -83,14 +83,14 @@
...
             @* It doesn't make sense to hide the first column (Target type), which lists
                 names of the types and members that have possible problems in porting. *@
             @for (int i = 1; i < detailColumnHeaders.Count; i++)
             {
                 var id = "checkbox" + i.ToString();
-                <input type="checkbox" id="@id" onchange='onToggleColumnVisibility(this)'>
+                <input type="checkbox" id="@id" aria-labelledby="hidecolumnsId @id" onchange='onToggleColumnVisibility(this)'>
```
Comment (scottaohara) [inline src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml:93]: remove aria-labelledby here

**Pull request merged**